src/layers/models/layersManager.ts

```diff
@@ -336,15 +336,20 @@ export class LayersManager {
   }
 
   private async validateJobNotRunning(productId: string, productType: ProductType): Promise<void> {
+    const notFoundIndex = -1;
     const findJobParameters: IFindJobsRequest = {
       resourceId: productId,
       productType,
       isCleaned: false,
       shouldReturnTasks: false,
     };
+    const ingestionJobTypes = this.config.get<string[]>('forbiddenTypesForParallelIngesion');
     const jobs = await this.db.getJobs<Record<string, unknown>, ITaskParameters | IMergeTaskParams>(findJobParameters);
     jobs.forEach((job) => {
-      if (job.status == OperationStatus.IN_PROGRESS || job.status == OperationStatus.PENDING) {
+      if (
+        job.status == OperationStatus.IN_PROGRESS ||
```

Comment:
This is a bug you need to check 
(job.status == OperationStatus.IN_PROGRESS || job.status == OperationStatus.PENDING) && ingestionJobTypes.indexOf(job.type) == notFoundIndex